Project/class_menu.rb

```diff
@@ -0,0 +1,246 @@
+# frozen_string_literal: true
+
+require_relative 'train.rb'
+require_relative 'station.rb'
+require_relative 'route.rb'
+require_relative 'passenger_train.rb'
+require_relative 'passengervagon.rb'
+require_relative 'cargovagon.rb'
+require_relative 'cargo_train.rb'
+require_relative 'main_menu.rb'
+require_relative 'class_menu.rb'
+class Menu
+  attr_accessor :alltrains, :routelist, :stationlist, :allvagons
```

Comment:
упрости имена
у тебя ненужные префиксы и постфиксы
```suggestion
  attr_accessor :trains, :routes, :stations, :vagons
```